### ISSUE TITLE
[Feature Request] Add support for IR/cursor from non-wiimote sources (mouse, Switch Pro Pad, etc.)

### ISSUE THREAD

**Issue by legend800** [cemu-project/Cemu#232]
Currently, there's no way to use another device for in-game IR other than a real wiimote. Only motion via dcu-client is supported, not cursor.

Please add native or dcu-client support for other controllers like a mouse, Switch Pro Pad, etc. to control IR.

Dolphin supports IR via motion from Switch pad as an example:
![2022-09-09 17_32_19-Wii Remote 1](https://user-images.githubusercontent.com/20983844/189461589-22d312a1-4c90-4442-9fe4-f8526c0b6898.png)

**Comment by MPS64** [comment]
agreed, been waiting for support on this for years and they still haven't done anything for it.

**Comment by Masamune3210** [comment]
For a large portion of those years, there wasn't a "they"

**Comment by MPS64** [comment]
> For a large portion of those years, there wasn't a "they"

That's true! I'm just very sad that they haven't been interested in implementing support for it, since they laughed me out of their discord for even suggesting it.

**Comment by AFrameWorker** [comment]
Yeah, can we have support for this already? Dying to be able to play Wii Party U properly again. Well, as 'properly' as I can do with a ps4 controller. If I can perfectly play Wii Party on Dolphin with my ps4 controller and its gyro, then heck I should be able to play Wii Party U doing so as well.

**Comment by GloriousGlider8** [comment]
Wii Party U just does not work w/out this feature and you can easily get softlocked.

**Comment by MPS64** [comment]
This is actually so hilarious- They still haven't added this yet? Will Wii Remotes continue to be "partially emulated" for the foreseeable future?

**Comment by hipoleto** [comment]
Please consider adding this, on dolphin you can use your mouse as the ponter or if you got for example a switch pro controller, the gyro simulates the wii pointer, Cemu really needs this as games like Wii Party U or Mario Party 10 requiere the pointer for some things, and without it you simply can get stuck on those games.

**Comment by OilFreak** [comment]
Yeah this is 100% a requirement for this emulator to be in a usable state for me. Seeing as I can't get it to work with a real wiimote, mouse support would be really appreciated. 
Also, it would be nice if you could aggregate multiple inputs into the emulated wiimote. If I can never get a real wiimote working, then the dream would be controller in one hand for the analog stick and mouse in the other

**Comment by PeterNjeim** [comment]
This feature is very much needed, but let's not artificially restrict it to motion or the mouse cursor, any controller buttons and joystick movements should be allowed to move the Wii remote pointer

**Comment by Arcadius2006** [comment]
i need this...... i need this for my piki3.........

**Comment by LorVC** [comment]
Upvote

**Comment by PeterNjeim** [comment]
@LorVC press the thumbs up button instead of making a comment. GitHub added reactions specifically for this reason, as making a comment sends a notification to everyone here, which in this case is considered spam. Thanks
